### ISSUE TITLE
Import from RSS

### ISSUE THREAD

**Issue by tiotrom** [z-------------/CPod#128]
I cannot seem to find this particular podcast librelounge.org and I was thinking if it is possible to import its RSS :)

**Comment by tiotrom** [comment]
Oh that's beautiful! Is that written somewhere in the documentation? Thank you!
By the way the titles, and thumbs are not picked up from the RSS
![2019-01-16_11-58](https://user-images.githubusercontent.com/34804533/51244928-99e8e580-1986-11e9-8746-aa63b295b3f5.png)
They work in Cantata though, with the same RSS file
![2019-01-16_12-00](https://user-images.githubusercontent.com/34804533/51244960-b38a2d00-1986-11e9-895a-1250fc628cd3.png)

**Comment by tiotrom** [comment]
I meant in cantata it works to pick up the name of the episodes but I don't think they have any thumb images

**Comment by tiotrom** [comment]
The titles are picked up in feebdurner as well https://feeds.feedburner.com/LibreLounge so I am thinking is something to do with CPod ;) - btw amazing application thank you very much!

**Comment by z-------------** [comment]
Thank you for pointing out those bugs. They have been fixed in c18f677. It's not written in the documentation, but it is pointed out in the hint for the search bar ("Search or enter feed URL"). Perhaps I can make the wording more clear in future releases.

**Comment by tiotrom** [comment]
Ah yes my bad for not noticing that. I think is very self-explanatory and it is just me who missed that :+1:

**Comment by tiotrom** [comment]
Just a follow-up question, you means you fixed the bug with titles and thumbnails but it was not pushed to the app for now? I am unsure how this works. Thanks!

**Comment by z-------------** [comment]
That is correct. The reason is that there is a half-finished feature in the code that I'm still working on so I want to get that done before pushing a new release. When that happens, you can either have pre-releases enabled in CPod's in-app Update settings and the update will automatically download the next time you start CPod, or you can directly download a new version from this GitHub repository's Releases section.

**Comment by tiotrom** [comment]
Ok thank you I just wanted to make sure I understood it correctly! :)
